pubspec.yaml

```diff
@@ -10,7 +10,7 @@ environment:
   sdk: '>=2.3.0 <3.0.0'
```

Comment:
also need to bump the min SDK support?

Reply:
OK, I'll try 2.7.0 and see if that satisfies Travis and the version solver. (The reason I bumped this is that there's no analyzer package that works on 2.3.0 and meets dart_style's new higher constraint.)